### DISCUSSION TITLE
GitHub pricing page has visual bug

### DISCUSSION THREAD

**Discussion by DagonWat** [community/community#11810] [Other Feature Feedback, Questions, & Ideas]
Hey Guys,

I was scrolling through pricing page and noticed one of the divs being overflow scroll (1 image), I think one of developers mistakenly added `overflow-x-scroll` css class. You can simply remove it and set `overflow-x: auto`, which will be better, since it will remove scrollbar (image 2).

![image](https://user-images.githubusercontent.com/18580766/154840911-f5ec7dd9-b530-4fd1-97c7-35d4d00a26aa.png)
![image](https://user-images.githubusercontent.com/18580766/154840823-9d71340a-9d22-4687-b087-8749aa955469.png)

**Comment by gaknoll** [comment]
Thanks @DagonWat for reporting this bug! I've shared it with the relevant team

**Comment by ajashams** [comment]
Yeah thanks @DagonWat -- I'm not sure how to convert this to an issue, but since this page is owned by #brand-design, it may be worth copying this over and [logging this as a bug](https://github.com/github/feedback/discussions/11810#discussion-3889441) 🙏

**Reply by DagonWat** [reply]
Hey there, @ajashams. Could you please share with me how that is done? Can't find any way to transfer issue to the other place.

**Comment by chalemen** [comment]
Todo lo ise por medio de un solo teléfono no use laptop ni table ni algo parecido mañana si gustan alas 14:00 horas me conectaré y veré si en verdad han ayudado a mi México querido mi país ya ucrania y alos que fueron afectados me da gusto que se parara laguerra!!Pero si algún país kiere o anuncia otra guerra no solo le quitaré su capital si no que dejaré limpia su tarjeta madre con millas de archivos para su país e infección a sus apps para que le sea difícil pararse y no me gaste nada de dinero de finanzas lo tengo todo en mi cuenta por si gustan mañana se los daré para apoyar alas personas con pocos recursos y lo siento por tantos problemas pero ya me canse de ver muertes a familias con un gran futuro por delante arruinado solo por querer mas poder eso no va y mucho gusto me llamo Carlos y soy de guadalajara jalisco y mañana doy mis calles y número de casa por si gusta pasar por mi para aveŕ si me contrata alguien para ayudar al futuro del mundo para ser mejor....yo tengo pocos recursos trabajo de guardia de seguridad orita jajajaja pero siempre e visto como crecer y espero ver al mundo crecer con migo y aki tienen todo mi apoyo aun que no sea mucho lo poco que tengo es tuyo y mio!¡!!
